src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -34,6 +34,7 @@ def load_arguments(self, _):
         c.argument('infrastructure', options_list=['--infrastructure'], help='The infrastructure on which the Kubernetes cluster represented by this connected cluster will be running on.', arg_type=get_enum_type(Infrastructure_Enum_Values))
         c.argument('disable_auto_upgrade', options_list=['--disable-auto-upgrade'], action='store_true', help='Flag to disable auto upgrade of arc agents.')
         c.argument('cl_oid', options_list=['--custom-locations-oid'], help="OID of 'custom-locations' app")
+        c.argument('onboarding_timeout', options_list=['--onboarding-timeout'], help='Time required (in minutes) for the arc-agent pods to be installed on the kubernetes cluster. Override this value if the hardware/network constraints on your cluster requires more time for installing the arc-agent pods.')
```

Comment:
seconds would be more standard time parameter?

Reply:
Should we group **"no_wait**" and **"onboarding_timeout"** parameters using _arg_group_? as they are both related to the wait time for onboarding of agent pods.

Reply:
updated to seconds. added arg grp